extensions/amp-next-page/1.0/visibility-observer.js

```diff
@@ -156,8 +169,106 @@ export default class VisibilityObserver {
   constructor(ampdoc) {
```

Comment:
Dima is working on a IntersectionObserver polyfill, which will make InOb available everywhere. Can this be easily extended to InObs?

Reply:
If it has the same `InsersectionObserver` behavior then no. The issue is when the element is bigger than the viewport, the `IntersectionObserver` stops serving updates whereas here we need to still be updated, hence this polyfill.